rootfs/bin/get_object

```diff
@@ -4,8 +4,9 @@ GET_PATH=slug.tgz
 
 export BUCKET_FILE=/var/run/secrets/deis/objectstore/creds/builder-bucket
 if [ "$BUILDER_STORAGE" == "minio" ]; then
+  MINION_BUCKET_NAME=${MINIO_BUCKET:-git}
```

Comment:
Oh my gosh! I was sleeping when I write this 😂 sorry!
I hope Jenkins get well soon :P